crates/core/tedge/src/cli/log.rs

```diff
@@ -319,6 +321,7 @@ impl fmt::Display for ConfigLogger<'_> {
         if let Some(mosquitto_version) = self.mosquitto_version {
             self.log_single_entry(f, "mosquitto version", &mosquitto_version)?;
         }
+        self.log_single_entry(f, "cryptoki", &self.cryptoki)?;
```

Comment:
It would be better to display this close to the auth type. And, even better, only when the `auth type` is  `Certificate`.

Reply:
fixed in 8f8a82a8b5